src/target/riscv/riscv.c

```diff
@@ -492,6 +488,10 @@ static int find_trigger(struct target *target, int type, bool chained, unsigned
 					/* Found num_required consecutive free triggers - success, done. */
 					if (idx)
 						*idx = i - (num_required - 1);
+					LOG_TARGET_DEBUG(
+						target,
+						"Trigger of type %d found on index %u, chained == %d",
+						type, i - (num_required - 1), chained);
```

Comment:
Minor thing: Here you can print `chained == true|false` instead of `1|0` to make it clearer what it really means.

```suggestion
					LOG_TARGET_DEBUG(
						target,
						"Trigger of type %d found on index %u, chained == %s",
						type, i - (num_required - 1), chained ? "true" : "false");
```

Reply:
Addressed.